Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -252,20 +252,23 @@ public class SQLiteBookmarksModelFactory: BookmarksModelFactory {
 }
 
 extension SQLiteBookmarks {
+    private static var isEditableQuery: String {
```

Comment:
Just put this at the top level as a `private let` two lines above, rather than jumping through the hoops to put it inside the extension.